cmd/create_test.go

```diff
@@ -28,6 +28,26 @@ var _ = Describe("Create template", func() {
 		})
 	})
 
+	Context("given .env.example", func() {
+		It("contains project name at DATABASE_URL", func() {
+			content := tests.ReadFile(".env.example")
+
+			fileContainProjectName := strings.Contains(content, "DATABASE_URL=postgres://postgres:postgres@0.0.0.0:5432/test-gin-templates_development?sslmode=disable")
+
+			Expect(fileContainProjectName).To(BeTrue())
```

Comment:
Just a small suggestion
```suggestion
			content := tests.ReadFile(".env.example")

			expectedContent := "DATABASE_URL=postgres://postgres:postgres@0.0.0.0:5432/test-gin-templates_development?sslmode=disable"

			Expect(content).To(ContainSubstring(expectedContent))
```

Reply:
Looks good. I will refactor this in https://github.com/nimblehq/gin-templates/issues/35.